scripts/icp/deploy-devnet.sh

```diff
@@ -107,14 +142,6 @@ echo "Done! Cycles transferred to context contract: $CONTEXT_ID"
 CONTEXT_ID=$(dfx canister id context_contract)
 LEDGER_ID=$(dfx canister id ledger)
 
-# Build contracts
-echo "Building contracts..."
-cd "$(dirname $0)"
```

Comment:
Yeah I'm not sure removing this `cd` is the right idea

Scripts dealing with relative paths should have better definition